java/test/jmri/util/swing/JemmyUtil.java

```diff
@@ -143,7 +143,7 @@ static public void waitFor(JmriJFrame f) {
                 return f.isActive();
             });
             count--;
-            f.requestFocus();
+            f.requestFocusInWindow();
```

Comment:
@bobjacobsen 
Can you look at this? I don't know much about Jemmy and since this might affect all JMRI it would be good if you could take a look.

Reply:
@danielb987 Thanks for checking carefully.  I think this is OK.  It selects a component for focus within the current window, which is fine because there's an earlier general `requestFocus` in the method.  Not sure why @DanielBoudreau did this, but it's OK with me.